app/lib/search_service.dart

```diff
@@ -98,6 +109,16 @@ class SearchService {
   }
 }
 
+// https://developers.google.com/custom-search/docs/structured_search#bias-by-attribute
+enum SearchBias { strong, weak }
+
+SearchBias parseExperimentalBias(String value) {
+  if (value == null) return null;
+  if (value == 'strong') return SearchBias.strong;
+  if (value == 'weak') return SearchBias.weak;
```

Comment:
Consider adding `'hard'` here as well, which seems to be the default, i.e. what we are using atm.